config/config.go

```diff
@@ -63,21 +64,117 @@ type DataStore struct {
 	DBNumber int
 }
 
+var defaultConfig = &config{
+	Munch: Munch{
+		Server: MunchServer{
+			Hostname: "localhost",
+			Port:     "50050",
+		},
+		LogLevel: "Logger",
+	},
+	DocumentStore: "mongo",
+	Mongo: DataStore{
+		Name:     "mongo",
+		URI:      "mongodb://localhost:27017",
+		DBName:   "meteomunch",
+		DBNumber: 0,
+	},
+	DLMRedis: DataStore{
+		Name:     "redis",
+		URI:      "redis://localhost:6379",
+		DBNumber: 1,
+	},
+	MeteoProviders: []MeteoProvider{
+		{
+			Name:    "open-meteo",
+			APIKey:  "",
+			BaseURI: "https://api.open-meteo.com/",
+		},
+	},
+}
+
 var legal = &struct {
 	documentStore []string
 }{
 	documentStore: []string{"mongo"},
 }
 
-// TODO: This can be more perfomant by loading it just once and pass a copy for each call
```

Comment:
Why remove this TODO? We are still executing this whole function for each call, right?